### PR TITLE
Register Mr.Unknown_74.is-a.dev

### PR DESCRIPTION
**Pull request by DynamiByte**
Register Mr.Unknown_74.is-a.dev with CNAME record pointing to mr-unknown-74.github.io.

### CODE REVIEW

File: domains/mr.unknown_74.json
Comment (21Z) [inline domains/mr.unknown_74.json:1]: Domain cannot contain underscore

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/mr.unknown_74.json
Comment (Stef-00012) [reply domains/mr.unknown_74.json:1]: they can contain underscores if they're in the middle

Comment (Stef-00012) [inline domains/mr.unknown_74.json:1]: the domain `unknown_74` doesn't exist so you can't create a subdomain of it

**Review by Stef-DP — CHANGES_REQUESTED** [review]
seems good to me

File: domains/mr.unknown_74.json
Comment (Stef-00012) [inline domains/mr.unknown_74.json:1]: the domain `unknown_74` doesn't exist so you can't create a subdomain of it (`mr`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**